src/expressions/functions/StringFunctions.ts

```diff
@@ -81,7 +81,7 @@ export function rTrim(original: Expr): CypherFunction {
  * @group Cypher Functions
  * @category String
  */
-export function split(original: Expr, delimiter: Expr): CypherFunction {
+export function spltest(original: Expr, delimiter: Expr): CypherFunction {
```

Comment:
typo?

Reply:
oh, great catch

This was a find/replace issue

Reply:
oh, it was already fixed :)
sorry for the confusion

Reply:
np!